vllm/envs.py

```diff
@@ -95,6 +95,7 @@
     VLLM_DP_MASTER_PORT: int = 0
     VLLM_MARLIN_USE_ATOMIC_ADD: bool = False
     VLLM_V0_USE_OUTLINES_CACHE: bool = False
+    VLLM_TPU_DISABLE_TOPK_TOPP_OPTIMIZATION: bool = False
```

Comment:
I don't think we need this. Can you add a follow up PR to remove this?

Reply:
Yes, will send a follow up PR to remove this once things look stable after merging.